### DISCUSSION TITLE
Passing the `draft` argument to `\includegraphics{}`?

### DISCUSSION THREAD

**Discussion by ramiromagno** [quarto-dev/quarto-cli#6234] [Q&A] [answered]
### Description

Is there a simple way of adding the `draft` option to:

https://github.com/ramiromagno/wiley-njd/blob/885a1827617952d3d01156ab68e6af8c8e118337/template.qmd#L131

such that instead of getting:

```tex
\begin{figure}[t]
{\centering \includegraphics[width=\textwidth,height=1.5in]{empty.pdf}}
\caption{\label{fig-fig1}This is the sample figure caption.}
\end{figure}
```

I would get the argument `draft` passed on to `\includegraphics{}`:

```tex
\begin{figure}[t]
{\centering \includegraphics[width=\textwidth,height=1.5in, draft]{empty.pdf}}
\caption{\label{fig-fig1}This is the sample figure caption.}
\end{figure}
```

**Comment by cderv** [comment, marked as answer]
Unfortunately, Pandoc does not support any attributes for LaTeX so this is why it is not passed if you pass `draft` as an Image Attribute. 

I don't think we support this in Quarto for now using the Markdown figure syntax. (https://quarto.org/docs/authoring/figures.html). 

I'll open an issue / FR about this.

**Reply by ramiromagno** [reply]
Cheers!

**Reply by ramiromagno** [reply]
I realized now this is a duplicate of https://github.com/quarto-dev/quarto-cli/discussions/5175.

**Comment by mcanouil** [comment]
Opened issue at https://github.com/quarto-dev/quarto-cli/issues/6239
